tests/test_data/test_ggtrends.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from epigraphhub.data import ggtrends
+
+
+def test_connect():
+    ggtrends._get_connection()
+    ggtrends._get_connection(language="ch-fr", timezone=60)
+
+
+def test_payload():
+    keywords = ["coronavirus", "covid"]
+    trends = ggtrends._build_payload(keywords)
+
+
+def test_historical_interest():
+    keywords = ["coronavirus", "covid"]
+    df = ggtrends.historical_interest(keywords)
+    assert len(df) > 0
```

Comment:
Assuming that it is a pandas dataframe, you can also use:

```suggestion
    assert not df.empty
```